src/pages/TRanimeizle/main.ts

```diff
@@ -1,12 +1,19 @@
 import { pageInterface } from '../pageInterface';
 
-function GetOverviewAnchor(): HTMLAnchorElement | null {
-  return document.querySelector(`a[href^="/anime"]`);
+function GetOverviewAnchor() {
+  const anchor = document.querySelector(`a[href^="/anime"]`) as HTMLAnchorElement;
+
+  if (!anchor) {
+    throw Error("Can't find overview anchor element");
+  }
+
+  return anchor;
 }
 
-function RemoveUnrelatedTurkishWordIzle(title: string) {
-  // "izle" is the translation of the word "watch"
-  return title.replace(/(?: |-)izle.*/i, '').replace(/(?: |-)İzle.*/i, '');
+function RemoveTurkishPhrases(title: string) {
+  // "izle" or "İzle" is the translation of the word "watch" or "Watch"
+  // Regexp can be test it out in here https://regex101.com/r/gULQnv/1
+  return title.replace(/(?: |-)[İi]zle.*/i, '');
```

Comment:
I have kind of reduce the excessive usage of two similar replace functions. This should satisfy the need.

There is also a link above to test the expression in with live samples